scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -116,6 +138,11 @@ func _moth_jump(delta: float) -> void:
 		super.apply_gravity(delta)
 	
 	
+func apply_climbing_movement(delta: float) -> void:
+	velocity.y = move_toward(velocity.y, _current_input_direction.y * CLIMB_SPEED, ACCELERATION * delta)
+	velocity.x = move_toward(velocity.x, _current_input_direction.x * MAX_SPEED, ACCELERATION * delta)
```

Comment:
Not sure if it matters but if I am moving too fast I cant grab it. If I launch myself with a super jump I cannot "stop" myself  on it. Not sure if controls should be more responsive on it or not